packages/playground/src/views/nodes.vue

```diff
@@ -1,11 +1,20 @@
 <template>
-  <div class="hint">
+  <!-- <div class="hint">
     <v-alert class="mb-4" type="info" variant="tonal">
       Node status is updated every 90 minutes. For a realtime status, click on the node's card.
     </v-alert>
-  </div>
+  </div> -->
```

Comment:
Do we really need to comment this out? I noticed that you moved it below, so let's delete it.